giraffe/src/utils/lineData.ts

```diff
@@ -51,3 +51,15 @@ export const simplifyLineData = (
 
   return result
 }
+
+export const getDomainDataFromLines = (
+  lineData: LineData,
+  domainLabel: string
+): Array<number> => {
+  let result = []
+  const numberOfLines = Object.keys(lineData).length
+  for (let index = 0; index < numberOfLines; index += 1) {
+    result = result.concat(lineData[index][domainLabel])
+  }
+  return result
+}
```

Comment:
would love to see even a simple test for this!

Reply:
Will add the tests with the next push.